### DISCUSSION TITLE
can be able to slove this problem since 2days help pls...

### DISCUSSION THREAD

**Discussion by Dico92** [community/community#54440] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Git giving this message everytime .......There was a problem with the editor ''C:/Program Files/Notepad++/notepad++.exe' -multiInst -notabbar -nosession -noPlugin'](https://stackoverflow.com/questions/73512012/there-was-a-problem-with-the-editor-c-program-files-notepad-notepad-exe)

**Comment by ghostinhershell** [comment]
Hi @Dico92, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, and give us more information about the error that you're seeing. Giving us a few more details might help someone give you a nudge in the right direction.
